root_VS2010/programs/C#/10_Build_WebApp_sample.bat

```diff
@@ -13,7 +13,7 @@ set ORG_PATH=%PATH%
 @rem --------------------------------------------------
 @rem Get the path to the executable file.
 @rem --------------------------------------------------
-set CURRENT_DIR=%~dp0
+set CURRENT_DIR="%~dp0"
```

Comment:
@yusukemaegawa 

Planned have changed by various circumstances.
Please check that the plan was modified as following.
https://github.com/OpenTouryoProject/OpenTouryoTemplates/issues/23#issue-123412442